shared_model/interfaces/query_responses/roles_response.hpp

```diff
@@ -31,10 +31,13 @@ namespace shared_model {
     class RolesResponse
         : public Primitive<RolesResponse, iroha::model::RolesResponse> {
      public:
+      /// type of roles collection
+      using RolesIdType = std::vector<types::RoleIdType>;
```

Comment:
We need some standart way of placing types. Because some of it is in types.hpp and some in class def

Reply:
Types.hpp contains only defs which are widely used when class defs are ones which used in 1-2 places, like this.